src/components/ComponentDetails/PropTable/PropDescription/index.tsx

```diff
@@ -1,13 +1,17 @@
-import { IcTypography } from "@ukic/react";
 import React from "react";
 import CodeAttribute from "../../../CodeAttribute";
+import "./index.css";
 
-const PropDescription: React.FC<{ description: string; type: string }> = ({
-  description,
-  type,
-}) => (
-  <div>
-    <IcTypography>{description}</IcTypography>
+const PropDescription: React.FC<{
+  description: string;
+  type: string;
+  required: boolean;
+}> = ({ description, type, required }) => (
+  <div className="prop-description">
+    <ic-typography>{description}</ic-typography>
+    {required && (
+      <ic-typography variant="subtitle-small">required</ic-typography>
```

Comment:
```suggestion
      <ic-typography variant="subtitle-small">Required</ic-typography>
```
Small change, but the designs show "Required" with a capital R
<img width="306" alt="Screenshot 2023-01-24 at 18 46 22" src="https://user-images.githubusercontent.com/113986285/214381386-6dc2ba61-7a2d-4aca-b1df-b53927d27750.png">